Mocks/Http/IMockedHttpClientOptions.cs

```diff
@@ -0,0 +1,26 @@
+﻿// ==++==
+//
+//   Copyright (c) Shadowsoft Corporation.  All rights reserved.
+//
+// ==--==
+
+namespace Yextly.Testing.Mocks.Http
+{
+    /// <summary>
+    /// Specifies options used to configure an <see cref="HttpClient"/> instance.
+    /// </summary>
+    public interface IMockedHttpClientOptions
+    {
+        /// <summary>
+        /// The delay that must be introduced for asynchronous calls.
+        /// </summary>
+        /// <remarks>The delay is always introduced before evaulating the expections.</remarks>
+        TimeSpan DefaultAsynchronousDelay { get; set; }
+
+        /// <summary>
+        /// The delay that must be introduced for synchronous calls.
+        /// </summary>
+        /// <remarks>The delay is always introduced before evaulating the expections.</remarks>
+        TimeSpan DefaultSynchronousDelay { get; set; }

```

Comment:
Why the two delays are separated? What is the rationale of having two sets of delays?

Reply:
Although the network layer should be the one, I thought to have them separated... but effectively, one should suffice